lib/rubocop/cop/rails/date.rb

```diff
@@ -68,12 +74,25 @@ def on_send(node)
 
           return if safe_chain?(node) || safe_to_time?(node)
 
+          check_deprecated_method(node)
+
           add_offense(node, location: :selector,
                             message: format(MSG_SEND, method: node.method_name))
         end
 
         private
 
+        def check_deprecated_method(node)
+          DEPRECATED_METHODS.each do |info|
```

Comment:
Ruby 2.0 introduced block keyword arguments, so we can do assignment there:

```ruby
DEPRECATED_METHODS.each do |relevant:, deprecated:|
  # relevant and deprecated are now available as local variables in this block
  ...
end
```

Reply:
Thanks! I've just updated the code.